msprime/simulations.py

```diff
@@ -1929,12 +1929,49 @@ def get_ll_representation(self):
 
 
 class BetaCoalescent(ParametricSimulationModel):
-    # TODO document.
+    """
+    A diploid Xi-coalescent with up to four simultaneous multiple mergers and
+    crossover recombination. The string ``"beta"`` can be used to refer to this
+    model.
+
+    There are two main differences between the Beta-Xi-coalescent and the
+    standard coalescent. Firstly, the number of lineages that take part in each
+    common ancestor event is random, with distribution determined by moments of
+    the Beta(`2 - alpha`, `alpha`)-distribution. All participating lineages are
+    randomly split into four groups, corresponding to the four parental
+    chromosomes in a diploid, bi-parental reproduction event. All lineages
+    within each group merge simultaneously.
+
+    Secondly, the timescale predicted by the Beta-Xi-coalescent is proportional
+    to `Ne^(alpha - 1)` generations. Note that it depends on `alpha`, and can
+    be dramatically shorter than the timescale of the standard coalescent.
+    Thus, effective population sizes must often be many orders of magnitude
+    larger than census population sizes. The per-generation recombination rate
+    is rescaled similarly to obtain the population-rescaled recombination rate.
+
+    For details on the distribution of the number of lineages taking part
+    in each common ancestor event, and on the associated timescale, see
+    Schweinsberg (2003) at
+    https://www.sciencedirect.com/science/article/pii/S0304414903000280.
+    Note however that the model of Schweinsberg (2003) is haploid, so that
+    all participating lineages merge in a common ancestor event without
+    splitting into four groups. The distributions of the number of
+    participating lineages in the haploid and diploid models are identical,
+    and the time scales differ only by a constant factor of 4.
+
+    The Beta-Xi-coalescent has two parameters. `alpha` determines the
+    degree of skewness in the offspring distribution, and must satisfy
+    `1 < alpha < 2`. `truncation_point` determines the maximum fraction
+    of the population replaced by offspring in one reproduction event.
+    It defaults to one for a standard Beta-Xi-coalescent, and must
+    satisfy `0 < truncation_point <= 1`. When `truncation_point < 1`,
+    then the distribution of the number of lineages participating in
+    a common ancestor event is determined by moments of the
+    Beta(`2 - alpha`, `alpha`)-distribution conditional on not
+    exceeding `truncation_point`.
+    """
     name = "beta"
 
-    # Must have 1 < alpha < 2 and 0 < truncation_point <= 1.
-    # alpha = 2 and truncation_point = 1 is equivalent to the Kingman coalescent,
-    # but numerical instability occurs with alpha > 2.0 - 0.0005.
     def __init__(self, reference_size=1, alpha=1.5, truncation_point=None):
```

Comment:
We should think carefully about these defaults now. I'm inclined to not set any, other than reference_size=1 to avoid mistakes. Unless you think these are really well chosen defaults @JereKoskela, let's follow this up in an issue.

Reply:
There is no obvious default for `alpha`, or for `psi` and `c` in the Dirac case. I'm happy to leave those without defaults. In contrast, I think `truncation_point = 1` is a good default, because it corresponds to the usual Beta-coalescent. It is almost certainly going to be what everyone who doesn't know to worry about it will want.